controller/search_spaces_blackbox_test.go

```diff
@@ -71,20 +72,81 @@ func (rest *TestSearchSpacesREST) TestSpacesSearchOK() {
 	idents, err := createTestData(rest.db, prefix)
 	require.NoError(rest.T(), err)
 	tests := []okScenario{
-		{"With uppercase fullname query", args{offset("0"), limit(10), prefix + "TEST_AB"}, expects{totalCount(1)}},
-		{"With lowercase fullname query", args{offset("0"), limit(10), prefix + "TEST_AB"}, expects{totalCount(1)}},
-		{"With uppercase description query", args{offset("0"), limit(10), "DESCRIPTION FOR " + prefix + "TEST_AB"}, expects{totalCount(1)}},
-		{"With lowercase description query", args{offset("0"), limit(10), "description for " + prefix + "test_ab"}, expects{totalCount(1)}},
-		{"with special chars", args{offset("0"), limit(10), "&:\n!#%?*"}, expects{totalCount(0)}},
-		{"with * to list all", args{offset("0"), limit(10), "*"}, expects{totalCountAtLeast(len(idents))}},
-		{"with multi page", args{offset("0"), limit(10), prefix + "TEST"}, expects{hasLinks("Next")}},
-		{"with last page", args{offset(strconv.Itoa(len(idents) - 1)), limit(10), prefix + "TEST"}, expects{hasNoLinks("Next"), hasLinks("Prev")}},
-		{"with different values", args{offset("0"), limit(10), prefix + "TEST"}, expects{differentValues()}},
+		{"With uppercase fullname query", args{offset("0"), limit(10), prefix + "TEST_AB", nil}, expects{totalCount(1)}},
+		{"With lowercase fullname query", args{offset("0"), limit(10), prefix + "TEST_AB", nil}, expects{totalCount(1)}},
+		{"With uppercase description query", args{offset("0"), limit(10), "DESCRIPTION FOR " + prefix + "TEST_AB", nil}, expects{totalCount(1)}},
+		{"With lowercase description query", args{offset("0"), limit(10), "description for " + prefix + "test_ab", nil}, expects{totalCount(1)}},
+		{"with special chars", args{offset("0"), limit(10), "&:\n!#%?*", nil}, expects{totalCount(0)}},
+		{"with * to list all", args{offset("0"), limit(10), "*", nil}, expects{totalCountAtLeast(len(idents))}},
+		{"with multi page", args{offset("0"), limit(10), prefix + "TEST", nil}, expects{hasLinks("Next")}},
+		{"with last page", args{offset(strconv.Itoa(len(idents) - 1)), limit(10), prefix + "TEST", nil}, expects{hasNoLinks("Next"), hasLinks("Prev")}},
+		{"with different values", args{offset("0"), limit(10), prefix + "TEST", nil}, expects{differentValues()}},
+
+		{
+			name: "sorted by name ascending",
+			args: args{
+				pageOffset: offset("0"),
+				pageLimit:  limit(10),
+				q:          prefix + "TEST",
+				sort:       sortPtr("name"),
```

Comment:
Please use `ptr.String` instead of `sortPtr`.

Reply:
done